docs/user/setting_up_your_development_environment.md

```diff
@@ -0,0 +1,101 @@
+# Setting up your development environment
+
+## Overview
```

Comment:
I think the following 3 sections can be covered by just linking to the installation tutorial which already exists.